### DISCUSSION TITLE
Maximum number of AudioPlayers?

### DISCUSSION THREAD

**Discussion by BobaTrek** [bluefireteam/audioplayers#1567] [Q&A]
Is there a location that lists the maximum number of AudioPlayers that can be instantiated simultaneously, and how many of those can be playing simultaneously?

Is the number different per target system?

**Comment by Gustl22** [comment]
As far as I know we didn't define any limit. But it would be good to have one stated (maybe). The only limit, I am aware of, is the 32 parallel streams in SoundPoolPlayer: https://github.com/bluefireteam/audioplayers/blob/516085a692624fa08d3d866b114ea6d25cc8b0e4/packages/audioplayers_android/android/src/main/kotlin/xyz/luan/audioplayers/player/SoundPoolPlayer.kt#L15
